src/qml/Legend.qml

```diff
@@ -133,15 +159,15 @@ ListView {
         width: rectangle.width
                - itemPadding
                - 46
-               - ( InTracking ? 34 : 0 )
-               - ( ( ReadOnly || GeometryLocked ) ? 34 : 0 )
-               - ( !IsValid ? 34 : 0 )
+               - (InTracking ? 29 : 0)
+               - ((ReadOnly || GeometryLocked) ? 29 : 0)
+               - (!IsValid ? 29 : 0)
```

Comment:
What are this calculations for? not to go under the symbols on the right side? Does it means the legend always has the same size?

Reply:
It's for the Text element (which needs a width to have elides) to not overlap with the legend badges